uint8 r_id
float64 x_layer1
float64 y_layer1
float64 x_prev_learned
float64 y_prev_learned
float64 what_ii
float64 conf_score

